src/index.js

```diff
@@ -4,6 +4,32 @@ const DEFAULT_WAIT_TIMEOUT = 500;
 const isFunction = (object) => typeof object === 'function';
 const isObject = (object) => typeof object === 'object';
 
+export const DATA_SPEC_ATTRIBUTE_NAME = 'data-spec';
```

Comment:
This part is imported from https://github.com/eventbrite/eventbrite_design_system/blob/master/src/utils/unitTest.js as is so as to avoid having a new peerDependency only for this snippet.
Anybody against it?